packages/dashboard-message-bus-e2e-test/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@truffle/dashboard-message-bus-e2e-test",
+  "version": "0.1.0-0",
+  "description": "Private e2e test package for testing the dashboard-message-bus and the dashboard-message-bus-client without circular dependencies",
+  "scripts": {
+    "test": "jest --verbose"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/trufflesuite/truffle/tree/master/packages/dashboard-message-bus-e2e-test#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/trufflesuite/truffle.git",
+    "directory": "packages/dashboard-message-bus-e2e-test"
+  },
+  "bugs": {
+    "url": "https://github.com/trufflesuite/truffle/issues"
+  },
+  "files": [
+    "dist"
+  ],
+  "directories": {
+    "test": "test"
+  },
+  "publishConfig": {
+    "access": "private"
+  },
```

Comment:
Change to `"private": true`? @benjamincburns 
https://github.com/trufflesuite/truffle/blob/555b886f1758aabd1e8f4dc42b3d76f12ee0a15c/packages/contract-tests/package.json#L2